packages/nodes-base/nodes/Aws/SQS/AwsSqs.node.ts

```diff
@@ -271,7 +272,7 @@ export class AwsSqs implements INodeType {
 					// loads first 1000 queues from SQS
 					data = await awsApiRequestSOAP.call(this, 'sqs', 'GET', `?Action=ListQueues`);
 				} catch (err) {
-					throw new Error(`AWS Error: ${err}`);
+					throw new NodeOperationError(this.getNode(), `AWS Error: ${err}`);
```

Comment:
don't know what it returns but gut feeling would say that this is rather an API Error than an Operation? what do you think?

Reply:
Good catch! It was an oversight. Fixed.